test/Common/DurableTaskHostExtensions.cs

```diff
@@ -96,5 +96,47 @@ internal static class DurableTaskHostExtensions
             IDurableOrchestrationClient client = clientRef[0];
             return client;
         }
+
+        /// <summary>
+        /// Helper function for the IDurableEntityClientBindingBackComp test. Gets an IDurableEntityClient.
+        /// </summary>
+        public static async Task<IDurableEntityClient> GetEntityClientBindingBackCompTest(
```

Comment:
nit: I realize you just copied the approach from previous tests, but since this logic is only used for one test, I think I prefer having it actually appear in the test instead of in a helper method. Otherwise just looking at the test doesn't give us a clear idea of what is happening.

Reply:
Sounds good, will fix and merge!